src/main/java/org/spongepowered/common/registry/type/effect/ParticleTypeRegistryModule.java

```diff
@@ -47,24 +47,22 @@
 import org.spongepowered.api.util.Direction;
 import org.spongepowered.common.effect.particle.SpongeParticleType;
 import org.spongepowered.common.item.inventory.SpongeItemStackSnapshot;
-import org.spongepowered.common.registry.AbstractCatalogRegistryModule;
+import org.spongepowered.common.registry.type.AbstractPrefixAlternateCatalogTypeRegistryModule;
 import org.spongepowered.common.registry.type.BlockTypeRegistryModule;
 import org.spongepowered.common.registry.type.ItemTypeRegistryModule;
 import org.spongepowered.common.registry.type.NotePitchRegistryModule;
 import org.spongepowered.common.registry.type.item.FireworkShapeRegistryModule;
 
-import java.util.Collections;
-import java.util.Locale;
-import java.util.Map;
+import java.util.*;
```

Comment:
please revert this, no wildcard imports
https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
Thanks. IntelliJ sniped me I need to fix my import settings ;)